tests/settings/ConfigurationGatheringTest.py

```diff
@@ -466,9 +466,9 @@ def test_get_filtered_bears(self):
             local_bears, global_bears = get_filtered_bears(
```

Comment:
@MacBox7 this is unsorted?

We need to apply a sort here I guess otherwise these will jiggle around randomly

Reply:
Either we can sort the output of `get_filtered_bears` or we can provide a utility in `TestUtilities.py`. Which one do you think would be best?

Reply:
Will it be a part of this PR?